### PR TITLE
 chore: revert commit for version 54.12.0

### PR DESCRIPTION
**Pull request by gbockus-sf**
This reverts commit ff49172c0534bebcd9941929ff70eb3d340efe80.

### What does this PR do?
revert build commit for moving to version v54.12.0
Needed in order to verify build via circleci for release.
Update commit builder script for change log to account for bad commit messages. 

This will enable us to re-run the release script in circle to generate the next release branch and change log.

### CODE REVIEW

File: scripts/change-log-generator-utils.js [hunk 1]
```diff
@@ -263,7 +263,7 @@ function filterPackageNames(packageHeaders) {
  */
 function generateKey(packageName, type, packagesToIgnore) {
   if (
-    typesToIgnore.some(typeToIgnore => type.startsWith(typeToIgnore)) ||
+    typesToIgnore.some(typeToIgnore => !type || type.startsWith(typeToIgnore)) ||
```
Comment (gbockus-sf) [inline scripts/change-log-generator-utils.js:266]: this is a work around to not fail to create the change log commit if we aren't able to parse a commit title.

File: docs/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "docs",
-  "version": "54.12.0",
```
Comment (gbockus-sf) [inline docs/package.json:3]: this was done via `git revert ff49172c0534bebcd9941929ff70eb3d340efe80`

Comment (floralan) [reply docs/package.json:3]: Just googled `git revert` vs `git reset`, and confirmed this is right commend we should use here. Good to learn more git knowledge from reviewing this PR!

**Review by floralan — APPROVED** [review]
Looks great!

File: docs/package.json (hunk 2 above)
Comment (floralan) [reply docs/package.json:3]: One more question I have: How would this `revert` enable us to re-run the release script in circle? I don't quite understand the connection between here and circle.

**Review by jeffb-sfdc — APPROVED** [review]

**Review by jeffb-sfdc — APPROVED** [review]

**Review by sbudhirajadoc — APPROVED** [review]

**Pull request merged**